mu4/scenes/inspector/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#=============================================================================
+#  MuseScore
+#  Music Composition & Notation
+#
+#  Copyright (C) 2020 MuseScore BVBA and others
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License version 2.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#=============================================================================
+get_filename_component(MODULE ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+message("Configuring " ${MODULE})
+
+set(MODULE_QRC ${CMAKE_CURRENT_LIST_DIR}/view/inspector_resources.qrc )
```

Comment:
I think it is better to move the resource file to the root of the module, as it is in other modules.

Reply:
Actually, I thought about it
But I prefer to leave it like this because I have not only the qml files, but also the widgets. So I think it's better to keep "views" in "views" folder

Reply:
Then the qml files will have even more nesting, which is already large.

Reply:
The inspector touches on a huge part of the functionality and it is logical that it is structured into sub-parts. Removing one level of nesting will not do much here, but will only break the order of the groupings, so I would like to leave it as it is.